platform/linuxbsd/display_server_x11.cpp

```diff
@@ -822,6 +825,24 @@ bool DisplayServerX11::screen_is_touchscreen(int p_screen) const {
 	return DisplayServer::screen_is_touchscreen(p_screen);
 }
 
+#ifdef DBUS_ENABLED
+void DisplayServerX11::screen_set_keep_on(bool p_enable) {
+	if (screen_is_kept_on() == p_enable)
+		return;
```

Comment:
Braces are now required for single-line `if` statements:

```suggestion
	if (screen_is_kept_on() == p_enable) {
		return;
	}
```